javascript/adoptbutton.js

```diff
@@ -0,0 +1,14 @@
+let Final = 0;
+
+function AddtoCart(value) {
```

Comment:
very small, but usually for function and variable names we use camel case. so start out with lowercase and capitalize each word after, so would be: `function addToCart(value)`